tools/pyang_plugins/gobgp.yang

```diff
@@ -549,6 +549,11 @@ module gobgp {
       type bmp-route-monitoring-policy-type;
       default PRE-POLICY;
     }
+    leaf statistics-timeout {
```

Comment:
It just follows Junos OS configuration name [`statistics-timeout`](https://www.juniper.net/documentation/en_US/junos/topics/reference/configuration-statement/statistics-timeout-edit-routing-options.html).
Also I picked the range for this value from Junos OS.
(Sorry I don't have strong will...)